app/lib/efile/dependent_eligibility/eligibility.rb

```diff
@@ -21,6 +21,7 @@ def benefit_amounts
             eip2: eip_two_eligibility.benefit_amount,
             eip3: eip_three_eligibility.benefit_amount,
             ctc: ctc_eligibility.benefit_amount
+            #add EITC?
```

Comment:
I guess I should remove this comment, but wanted a read on if this should be added to this list. It seems this list is only used for calculating the total_benefit_amount for a dependent which right now is only used in a test

Reply:
But I guess we don't calculate EITC on a per dependent basis more on the whole so I'm thinking we would leave it out of here